spec/wings/valkyrie/query_service_spec.rb

```diff
@@ -5,31 +5,35 @@
 
 RSpec.describe Wings::Valkyrie::QueryService do
   before do
-    class Book < ActiveFedora::Base
-      include Hyrax::WorkBehavior
-      include Hydra::AccessControls::Permissions
-      property :title, predicate: ::RDF::Vocab::DC.title, multiple: true
-      property :a_member_of, predicate: ::RDF::URI.new('http://www.example.com/a_member_of'), multiple: true do |index|
-        index.as :symbol
+    module Hyrax::Test
+      module QueryService
+        class Book < ActiveFedora::Base
+          include Hyrax::WorkBehavior
+          include Hydra::AccessControls::Permissions
+          property :title, predicate: ::RDF::Vocab::DC.title, multiple: true
+          property :a_member_of, predicate: ::RDF::URI.new('http://www.example.com/a_member_of'), multiple: true do |index|
+            index.as :symbol
+          end
+          property :an_ordered_member_of, predicate: ::RDF::URI.new('http://www.example.com/an_ordered_member_of'), multiple: true
+        end
+
+        class Image < ActiveFedora::Base
+          include Hydra::AccessControls::Permissions
+          property :title, predicate: ::RDF::Vocab::DC.title, multiple: true
+        end
       end
-      property :an_ordered_member_of, predicate: ::RDF::URI.new('http://www.example.com/an_ordered_member_of'), multiple: true
-    end
-    class Image < ActiveFedora::Base
-      include Hydra::AccessControls::Permissions
-      property :title, predicate: ::RDF::Vocab::DC.title, multiple: true
     end
   end
 
   after do
-    Object.send(:remove_const, :Book)
-    Object.send(:remove_const, :Image)
+    Hyrax::Test.send(:remove_const, :QueryService)
```

Comment:
I like this!  Makes for easier removal of constants defined in tests.

Reply:
yeah, we should probably start using Hyrax::Test everywhere test constants are needed